fastplotlib/layouts.py

```diff
@@ -133,6 +133,8 @@ def __getitem__(self, index: Union[Tuple[int, int], str]):
             for subplot in self._subplots.ravel():
                 if subplot.name == index:
                     return subplot
+                else:
```

Comment:
nope this will always raise after the first iteration, you need just a raise after the loop